master/internal/api_auth.go

```diff
@@ -124,23 +124,23 @@ func processProxyAuthentication(c echo.Context) (done bool, err error) {
 		return true, redirectToLogin(c)
 	}
 
-	taskID := c.Param("service")
-	ownerID, err := db.GetCommandOwnerID(c.Request().Context(), model.TaskID(taskID))
```

Comment:
Does it make sense to replace other instances of `GetCommandOwnerID` with `GetCommandGenericSpec`? If I'm understanding correctly, I think we could just get rid of `GetCommandOwnerID`

Reply:
That's a good point and I thought about that a bit but I think `GetCommandOwnerID` has to be a strictly smaller, albeit small, database hit so I opted to not do it since it didn't seem to benefit readability and or maintainability much

Reply:
I'll go ahead an land this now. I'd be happy to update this part in another pr.